model/qti/XIncludeLoader.php

```diff
@@ -133,7 +133,7 @@ private function parseCustomElementMarkup(CustomInteraction $customElement)
             throw new ParsingException('cannot parse pci markup');
         }
 
-        $customElement->setMarkup($xml->saveXML(null, LIBXML_NOEMPTYTAG));
+        $customElement->setMarkup($xml->saveXML($xml->documentElement, LIBXML_NOEMPTYTAG));
```

Comment:
Also, why we need to save it even if the `if ($loadSuccess && !is_null($node)) {` is false?
Maybe we can add a check if `$xincludesNodes->count()` then do some things and save it.
What do you think?

Reply:
just tested and if the package does not contain shared-stimulus or pci , it will not trigger that path of the code,
and while verifying the value of the `$xincludesNodes->count()` it is allays =